src/internal/m365/onedrive/collection.go

```diff
@@ -523,6 +542,7 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 					id:   itemID + dataSuffix,
 					data: itemReader,
 					info: itemInfo,
+					// extensionData: eh,
```

Comment:
Since `eh` won't be initialized until the lazy reader init runs, can't store a reference to it to run `GetExtensionData` during `FinishedFile`